domain-ee/ee-ep-merge-app/src/python_src/service/ep_merge_machine.py

```diff
@@ -490,6 +505,9 @@ def make_request(
     def has_error(self) -> bool:
         return self.job.state == JobState.COMPLETED_ERROR  # type: ignore[no-any-return]
 
+    def is_aborted(self) -> bool:
+        return self.job.state == JobState.ABORTED  # type: ignore[no-any-return]
```

Comment:
Oh, I see why the ignore comment is here, to placate the pedantry of `__eq__` potentially returning non-bool. Maybe it's better to just `bool()`ify the value? (I got the idea from this [SO post](https://stackoverflow.com/questions/75864807/mypy-complaining-about-no-any-return-rule-being-violated-for-obvious-boolean-e).)